README.md

```diff
@@ -60,7 +83,50 @@ that. When you are done, quit Minecraft.
 Go to your Minecraft folder, and run `$DS/scripts/update-journeymap.sh
 get-from-journeymap` to copy the latest map data into the git repo from your
 journeymap installation. (If your world is not named `Wynncraft`, you need to
-override )`WYNNCRAFT_WORLD_NAME`)
+override `WYNNCRAFT_WORLD_NAME`)
 
 Then go to `$DS`, do `git status` to verify that you have changed only the tiles
-in `worldmap/journeymap-data`, and then commit and push your changes.
+in `worldmap/journeymap-data`. Now comes the tricky part. Journeymap will always
+update all tiles you have visited, even if there are no visible changes. If we
+blindly push all binary changes in this directory, the repository will quickly
+grow in size. So, please double check all changed image tiles if you really
+intended to update them. If you were just about remapping a specific area,
+revert all changes except for that area. If you can't visibly spot any
+difference in unrelated areas, just revert those files.
+
+Finally, commit the real changed tiles, and push it upstream/open a PR.
+
+## Generating maps from collected tiles
+
+The next step is to use the scripts available here to generate the updated map
+png files and associated metadata. Make sure you have updated the tiles properly
+first as described above.
+
+### Artemis
+
+Run the shell script `scripts/update-maps-artemis.sh`. This will create a
+directory `worldmap/out` (and also other data files). Do not commit these
+generated files to the repository! Instead, copy them to the `maps` directory in
+where you have checked out https://github.com/Wynntils/WynntilsWebsite-API
+(called `$WWAPI` below). Note especially the updated maps.json, which contain
+new md5 sums for these maps.
+
+However **don't commit these files blindly** to `$WWAPI`! The script will
```

Comment:
Can't we somehow compare files in the script?

Reply:
Another way for this to get rid of all the unnecessary map changes is to temporarily delete all the old `do_map` commands and only keep the newly mapped command, then manually merging `maps.json`

Reply:
Maybe. It might also be not needed anymore. With the png compression I have added, unneeded metadata is stripped from png:s, and also timestamps are set to 2000-01-01 00:00:00 GMT automatically, all the time. So identical images should be identical byte-by-byte. I need to do some more testing to see if the tiles still contain random changes, but hopefully this will make sure that you can't change files without changing them.

Reply:
Also, @Incompleteusern has a good point. I often do that myself, at least commenting out main map, to save time.

It can perhaps be automated better, but, well, I don't know how much time to spend on making the tooling perfect. The main goal for me is reproducability. You should be able to run them and get consistent output.

Reply:
I'd like to leave this as it is for now. @kristofbolyai Is that okay?

Reply:
Okay, sure